packages/element-library/package.json

```diff
@@ -49,6 +49,7 @@
     "@googleforcreators/stickers": "*",
     "@googleforcreators/transform": "*",
     "@googleforcreators/units": "*",
+    "@googleforcreators/story-editor":  "*",
```

Comment:
This seems the wrong way around. Maybe it makes more sense to move useFont to it's own package and load that?

Reply:
Maybe I can use it higher up or something --- it would be nice to not pull it into the element-library

Reply:
So I think the question becomes for anything that pull props from the element 

i.e. in this case back up to 

```<DisplayElement```

Do we want to manipulate the element on the way in and or add font props with the extra data and follow that through the other code ... 

Or --- try to get the needed data at the part of the code that needs it for instance `calcFontMetrics` and `generateParagraphTextStyle`.

Reply:
@swissspidy Thoughts?

Reply:
Could you please elaborate on why this dependency was added/needed? What are we trying to achieve?

Theoretically adding it would be OK, but typically we've been passing additional information via props. See `getProxiedUrl`, `renderResourcePlaceholder`, `siblingCount`, `previewMode` as prior art.

Reply:
The original intention was looking for a way to get access to the data without having to pass it through several components ... looking further I don't think that's the right approach.

There many cases where we pass `element` and it hard to tell which of these components actually end up using any of the font data.

https://github.com/GoogleForCreators/web-stories-wp/blob/main/packages/element-library/src/text/output.js#L102

So we end up removing from element by re-adding to get the data to a nested function.


If this is about storage --- 
Started wondering if it would make more sense remove the data when saved and re-add (as needed when things load).